frontend/src/components/loginPrompt/index.js

```diff
@@ -115,30 +92,29 @@ class LoginPrompt extends Component {
 
     if (this.state.remainingTime && this.remainingTimeInterval) return
 
+    this.captchaRef.current.execute()
+  }
+
+  setElId = (item, id) => {
+    this.elId[item.props.name] = id
+  }
+
+  handleVerificationSuccess(token) {
     this.setState({
       username: 'loading',
       password: 'loading',
       loadingTimeout: true,
     })
 
-    if (!this.state.loadingTimeout) {
-      // this.recaptchaRef.current.reset()
-      // this.recaptchaRef.current.execute()
-    }
-  }
-
-  setElId = (item, id) => {
-    this.elId[item.props.name] = id
+    this.auth(token)
   }
 
-  onChange = (recaptchaToken) => {
-    this.setState({ recaptchaToken })
-    this.auth()
+  onCaptchaError = () => {
+    this.props.store.notification.setContent(popUpData.messages.captchaError)
```

Comment:
```suggestion
    const { notification } = this.props.store
    const { captchaError } = popUpData.messages

    notification.setContent(captchaError)
```

More readable and less chaining